fuse_optimizers/include/fuse_optimizers/batch_optimizer.hpp

```diff
@@ -147,30 +151,49 @@ class BatchOptimizer : public Optimizer
    */
   using TransactionQueue = std::multimap<rclcpp::Time, TransactionQueueElement>;
 
-  fuse_core::Transaction::SharedPtr combined_transaction_;  //!< Transaction used aggregate
-                                                            //!< constraints and variables from
-                                                            //!< multiple sensors and motions models
-                                                            //!< before being applied to the graph.
-  std::mutex combined_transaction_mutex_;  //!< Synchronize access to the combined transaction
-                                           //!< across different threads
+  // Read-only after construction
```

Comment:
I'm doing a bit of variable rearranging. This is not strictly required for this patch, but I wanted to mirror the FixedLagSmoother pattern in the BatchOptimizer.